From khali@linux-fr.org Mon Feb 27 14:12:07 2006
Date: Mon, 27 Feb 2006 23:12:58 +0100
From: Alessandro Zummo <a.zummo@towertech.it>
To: Greg KH <greg@kroah.com>
Cc: Alessandro Zummo <a.zummo@towertech.it>
Subject: I2C: i2c-ixp4xx: Add hwmon class
Message-Id: <20060227231258.cebdda2f.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-ixp4xx-hwmon-class.patch

From: Alessandro Zummo <a.zummo@towertech.it>

This patch adds I2C_CLASS_HWMON to the ixp4xx bus driver.

Signed-off-by: Alessandro Zummo <a.zummo@towertech.it>
Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/i2c/busses/i2c-ixp4xx.c |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/drivers/i2c/busses/i2c-ixp4xx.c
+++ gregkh-2.6/drivers/i2c/busses/i2c-ixp4xx.c
@@ -126,6 +126,7 @@ static int ixp4xx_i2c_probe(struct platf
 	drv_data->algo_data.timeout = 100;
 
 	drv_data->adapter.id = I2C_HW_B_IXP4XX;
+	drv_data->adapter.class = I2C_CLASS_HWMON;
 	strlcpy(drv_data->adapter.name, plat_dev->dev.driver->name,
 		I2C_NAME_SIZE);
 	drv_data->adapter.algo_data = &drv_data->algo_data;
